### PR TITLE
docstring_example_added_ivy_functional_ivy_activations_sigmoid

### LINKED ISSUE #18127: sigmoid

### PR DESCRIPTION
**Pull request by hemrajat**
Close #18127

### CODE REVIEW

**Comment by hemrajat** [comment]
Hi @hmahmood24 Can you please tell me what is the issue in failing checks ?
Some time a test fail and some time it passes

**Comment by hemrajat** [comment]
Hi @hmahmood24 Can you please tell me what is the issue in failing checks ?
Some time a test fail and some time it passes

**Comment by hmahmood24** [comment]
> Hi @hmahmood24 Can you please tell me what is the issue in failing checks ? Some time a test fail and some time it passes

Hey @hemrajat. Sure thing I'll review it today. Thanks :)

**Comment by hmahmood24** [comment]
Hey @hemrajat. Regarding the tests, you need not worry since you simply added/modified the docstring and not the actual code itself so the only test relevant to you is the `lint / Check formatting / check-formatting (pull_request)` workflow which is passing :) 

However, I reviewed your PR but honestly I can't find anything that's missing with the docstring that this PR strives to add. I think the docstring as is, is quite clear and succinct. Still I think I'm going to leave the decision with @KareemMAX if he thinks we should merge the PR based on the current content or not. Thanks anyways :)

**Comment by hemrajat** [comment]
Hi @hmahmood24 @KareemMAX  I got email from recruiting team given a list of todo there I saw this function in open todo #754. I saw some examples in doc string examples. I read in docstring example docs https://unify.ai/docs/ivy/overview/deep_dive/docstring_examples.html#ivy-add which says that example in docstring should cover all combination of arguments. Here in this PR I included one or two examples where out parameters is given, in earlier docstring example out argument is not passed. I strived to included that example. 
Thank you

**Comment by hemrajat** [comment]
Hi @hmahmood24 @KareemMAX Can you please review the pr ? I've provided the reason for creating this pr in previous comment. Please let me know if anything is not clear from my side or if I'm missing something

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @hemrajat 

I agree with the changes you made, though I have some comments that I'd love you address before merging the PR.

Thank you for the sharp eye and for your contribution 🚀🚀 Waiting for more contributions from you to ivy.

File: ivy/functional/ivy/activations.py [hunk 1]
```diff
@@ -437,15 +439,24 @@ def sigmoid(
...
-    ivy.array([[0.214, 0.978, 0.891], [0.846,0.985,0.001]] )
+    ivy.array([[0.214, 0.978, 0.891],
+               [0.846,0.985,0.001]])
+
+    >>> x = ivy.array([[-1.3, 3.8, 2.1],
+    ...                [1.7, 4.2, -6.6]])
+    >>> ivy.sigmoid(x,out=x)
+    >>> print(x)
+    ivy.array([[0.214, 0.978, 0.891],
+               [0.846,0.985,0.001]])
```
[the remark this answers is not in the document]
Comment (hemrajat) [reply ivy/functional/ivy/activations.py]: Hi @KareemMAX I've removed this section and added ivy.Container example. Please review the latest change thank you.

File: ivy/functional/ivy/activations.py [hunk 2]
```diff
@@ -418,14 +418,16 @@ def sigmoid(
     x
         input array.
     out
-        optional output array, for writing the result to. It must have a shape that the
-        input broadcast to.
+        optional output array. If output array is provided then result
+        of sigmoid will be stored in output array instead of creating new
+        array for storing the result, shape of x and out must be same.
```
[the remark this answers is not in the document]
Comment (hemrajat) [reply ivy/functional/ivy/activations.py]: Hi @KareemMAX I've adjusted the out description to previous one. For standardized purpose it is better to stick with previous description. 
Thank you

**Comment by hemrajat** [comment]
Hi @KareemMAX @hmahmood24 can you please review the changes I've made after resolving the comment of @KareemMAX ? 
Thank you

**Review by KareemMAX — APPROVED** [review]
LGTM! I have added a small clarification that we are using `ivy.Container` on the second half

**Pull request merged**